kubernetes/annotations_and_labels.md

```diff
@@ -10,47 +12,80 @@
   this annotation instead of having it in the spec. As this is an implementation
   detail for our operators.
 
-## Common Labels (for control plane services)
+## Common Labels
 
 - `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This should
-  also be used as replicas selector. E.g. `app=kvm-operator`.
-- `giantswarm.io/cluster` - value should contain tenant cluster ID which this
-  object is part of. E.g. `giantswarm.io/cluster=eggs2`.
+  to every Kubernetes resource associated with an application. This should also
+  be used as replicas selector. E.g. `app=kvm-operator`. Exceptions can be made
+  to accomodate for adherence to existing selectors upstream.
 - `giantswarm.io/certificate` - value should contain certificate name as
   defined in github.com/giantswarm/certs repo. This is used in certificate
   Secrets and CertConfigs.
+- `giantswarm.io/cluster` - value should contain tenant cluster ID which this
+  object is part of. E.g. `giantswarm.io/cluster=eggs2`.
 - `giantswarm.io/managed-by` - value should contain repository name of the
   operator managing the object. E.g. `giantswarm.io/managed-by=kvm-operator`.
 - `giantswarm.io/organization` - value should contain tenant cluster's
   organization ID as displayed in the front-end
-  `giantswarm.io/organization=track-hunter`.
+  `giantswarm.io/organization=adidas`.
```

Comment:
This is the only real change I made. I replaced that with adidas because now we have public customers. Other stuff is just moved around untouched.

Reply:
Ah yes this is actually created by our customer. Yes I think `dev` is good example.